rules_poetry/poetry.bzl

```diff
@@ -79,8 +94,31 @@ def _mapping(repository_ctx):
     return {
         "dependencies": dependencies,
         "groups": groups,
+        "pyproject": pyproject
     }
 
+def extract_markers(repository_ctx, resolved_markers, dep, markers):
+    python_interpreter = _resolve_python_interpreter(repository_ctx)
+
+    if str(type(markers)) == 'list':
```

Comment:
Can we fail if markers isn't a list? That way we can catch cases where markers isn't a list.

Reply:
its either a version string or a markers list for example:

```
networkx: "1.2.3"
```

or 

```
networkx: [
     {"version": "1.2.1", "markers": "platform_machine == 's390x'"},
     {"version": "1.2.3", "markers": "platform_machine != 's390x'"},
]
```